web/domain/persistence/dao.py

```
@@ -0,0 +1,261 @@
+        db.close()
+
+
+@tool("save_disclosure_info", args_schema=DisclosureList)
+def save_disclosure_info(
+        company_id: int,
+        rcept_no: str,
+        report_nm: str,
+        rcept_dt: datetime,
+        rpt_type: Optional[str] = None,
+        flr_nm: Optional[str] = None,
+        id: Optional[int] = None
```

Ruff (0.14.5)

Unused function argument: `id`

(ARG001)